src/Sitemap.php

```diff
@@ -62,9 +62,11 @@ public function hasUrl(string $url): bool
         return (bool) $this->getUrl($url);
     }
 
-    public function render(): string
+    public function render($sortAlphabetically = true): string
```

Comment:
Instead of adding an argument, add a chainable method `sortAlphabetically`. Also, add a test and update the readme.

Reply:
The current default behaviour is to sort alphabetically. If I add a separate chainable method then that will mean that everyone who upgrades will no longer be sorting their sitemap automatically unless they add the additional `->sortAlphabetically()` correct? This is not a breaking change per se because the order doesn't actually matter, but it will change the current behaviour somewhat.

@freekmurze You would be ok with this change then? I'd rather ask before I do them.

Reply:
I'm also a bit confused as to what the test should be testing since nothing is really changing and the order doesn't matter.

Reply:
It's seems like the current behaviour is not sorting alphabetically This PR changes that in line 68 of the `render` function. I only want that to happen when `->sortAlphabetically()` is being called.

The test could test that by default links aren't sorted and that when you call `sortAlphabetically` they are being sorted.

Reply:
@freekmurze 

There seems to be some confusion here. The current behaviour IS sorting the sitemap entries alphabetically because of this line here: https://github.com/spatie/laravel-sitemap/blob/master/src/Sitemap.php#L67

A previous PR had been opened to remove this unnecessary and unwanted sorting but you had rejected it here: https://github.com/spatie/laravel-sitemap/pull/209

With this PR I was hoping to once again propose that we do NOT sort alphabetically because it's unneeded and IMO it's confusing and no one really wants this, instead we should have control whether it's sorted or not.

That being said, what I wrote above still stands. Ideally we should just remove this line https://github.com/spatie/laravel-sitemap/blob/master/src/Sitemap.php#L67 and call it a day.

Your thoughts @freekmurze ?

Regards

Reply:
Thanks for clarifying. Seems like I needed a little bit more coffee this morning 🙂 

Meanwhile I tagged a new version where the sorting is removed. 👍

Reply:
Fantastic... I look forward to being able to use it in a few months when I'm able to upgrade to PHP 8 =D